File: ClipMobile/src/main/java/com/migueljteixeira/clipmobile/ui/StudentNumbersActivity.java
package com.migueljteixeira.clipmobile.ui;

import android.os.Bundle;

//import com.crashlytics.android.Crashlytics;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;

import com.migueljteixeira.clipmobile.R;

public class StudentNumbersActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_singlepane);

        setupActionBar();

//        Crashlytics.log("StudentNumbersActivity - onCreate");

        FragmentManager fm = getSupportFragmentManager();
        Fragment fragment = fm.findFragmentById(R.id.content_frame);
        if (fragment == null) {
            fragment = new StudentNumbersFragment();
            fm.beginTransaction().add(R.id.content_frame, fragment).commit();
        }
    }

}

File: ClipMobile/src/main/java/com/migueljteixeira/clipmobile/ui/StudentNumbersActivity.kt
package com.migueljteixeira.clipmobile.ui

import com.migueljteixeira.clipmobile.ui.BaseActivity
import android.os.Bundle
import com.migueljteixeira.clipmobile.R
import com.migueljteixeira.clipmobile.ui.StudentNumbersFragment

class StudentNumbersActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_singlepane)
        setupActionBar()

//        Crashlytics.log("StudentNumbersActivity - onCreate");
        val fm = supportFragmentManager
        var fragment = fm.findFragmentById(R.id.content_frame)
        if (fragment == null) {
            fragment = StudentNumbersFragment()
            fm.beginTransaction().add(R.id.content_frame, fragment).commit()
        }
    }
}